pysmt/factory.py

```diff
@@ -441,6 +441,31 @@ def get_model(self, formula, solver_name=None, logic=None):
                 retval = solver.get_model()
             return retval
 
+    def get_implicant(self, formula, solver_name=None,
+                      logic=None):
+        mgr = self.environment.formula_manager
+        if logic == AUTO_LOGIC:
+            logic = get_logic(formula, self.environment)
+
+        with self.Solver(name=solver_name, logic=logic) \
+             as solver:
+            solver.add_assertion(formula)
+            check = solver.solve()
+            if not check:
+                return None
+            else:
```

Comment:
I was thinking more of a static method (although it does not need to be static, we can just require that no assertion has been done yet). In my opinion, factory should only wrap methods of Solver, providing good defaults. 
However, this is not particularly blocking. If you see a nice way to do this, go ahead. Otherwise, we can do it later.